Fix building with C++14, which errors out due to narrowing conversions.
See also: https://bugs.gentoo.org/show_bug.cgi?id=592442

Patch by Francesco Turco

--- a/src/gtk/bdiag.xbm
+++ b/src/gtk/bdiag.xbm
@@ -1,6 +1,6 @@
 #define bdiag_width 16
 #define bdiag_height 16
 static char bdiag_bits[] = {
-   0x80, 0x80, 0x40, 0x40, 0x20, 0x20, 0x10, 0x10, 0x08, 0x08, 0x04, 0x04,
-   0x02, 0x02, 0x01, 0x01, 0x80, 0x80, 0x40, 0x40, 0x20, 0x20, 0x10, 0x10,
-   0x08, 0x08, 0x04, 0x04, 0x02, 0x02, 0x01, 0x01};
+   (char)0x80, (char)0x80, (char)0x40, (char)0x40, (char)0x20, (char)0x20, (char)0x10, (char)0x10, (char)0x08, (char)0x08, (char)0x04, (char)0x04,
+   (char)0x02, (char)0x02, (char)0x01, (char)0x01, (char)0x80, (char)0x80, (char)0x40, (char)0x40, (char)0x20, (char)0x20, (char)0x10, (char)0x10,
+   (char)0x08, (char)0x08, (char)0x04, (char)0x04, (char)0x02, (char)0x02, (char)0x01, (char)0x01};
--- a/src/gtk/cdiag.xbm
+++ b/src/gtk/cdiag.xbm
@@ -1,6 +1,6 @@
 #define cdiag_width 16
 #define cdiag_height 16
 static char cdiag_bits[] = {
-   0x81, 0x81, 0x42, 0x42, 0x24, 0x24, 0x18, 0x18, 0x18, 0x18, 0x24, 0x24,
-   0x42, 0x42, 0x81, 0x81, 0x81, 0x81, 0x42, 0x42, 0x24, 0x24, 0x18, 0x18,
-   0x18, 0x18, 0x24, 0x24, 0x42, 0x42, 0x81, 0x81};
+   (char)0x81, (char)0x81, (char)0x42, (char)0x42, (char)0x24, (char)0x24, (char)0x18, (char)0x18, (char)0x18, (char)0x18, (char)0x24, (char)0x24,
+   (char)0x42, (char)0x42, (char)0x81, (char)0x81, (char)0x81, (char)0x81, (char)0x42, (char)0x42, (char)0x24, (char)0x24, (char)0x18, (char)0x18,
+   (char)0x18, (char)0x18, (char)0x24, (char)0x24, (char)0x42, (char)0x42, (char)0x81, (char)0x81};
--- a/src/gtk/cross.xbm
+++ b/src/gtk/cross.xbm
@@ -1,6 +1,6 @@
 #define cross_width 15
 #define cross_height 15
 static char cross_bits[] = {
-   0x84, 0x10, 0x84, 0x10, 0xff, 0x7f, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10,
-   0x84, 0x10, 0xff, 0x7f, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10,
-   0xff, 0x7f, 0x84, 0x10, 0x84, 0x10};
+   (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0xff, (char)0x7f, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10,
+   (char)0x84, (char)0x10, (char)0xff, (char)0x7f, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10,
+   (char)0xff, (char)0x7f, (char)0x84, (char)0x10, (char)0x84, (char)0x10};
--- a/src/gtk/fdiag.xbm
+++ b/src/gtk/fdiag.xbm
@@ -1,6 +1,6 @@
 #define fdiag_width 16
 #define fdiag_height 16
 static char fdiag_bits[] = {
-   0x01, 0x01, 0x02, 0x02, 0x04, 0x04, 0x08, 0x08, 0x10, 0x10, 0x20, 0x20,
-   0x40, 0x40, 0x80, 0x80, 0x01, 0x01, 0x02, 0x02, 0x04, 0x04, 0x08, 0x08,
-   0x10, 0x10, 0x20, 0x20, 0x40, 0x40, 0x80, 0x80};
+   (char)0x01, (char)0x01, (char)0x02, (char)0x02, (char)0x04, (char)0x04, (char)0x08, (char)0x08, (char)0x10, (char)0x10, (char)0x20, (char)0x20,
+   (char)0x40, (char)0x40, (char)0x80, (char)0x80, (char)0x01, (char)0x01, (char)0x02, (char)0x02, (char)0x04, (char)0x04, (char)0x08, (char)0x08,
+   (char)0x10, (char)0x10, (char)0x20, (char)0x20, (char)0x40, (char)0x40, (char)0x80, (char)0x80};
--- a/src/gtk/horiz.xbm
+++ b/src/gtk/horiz.xbm
@@ -1,6 +1,6 @@
 #define horiz_width 15
 #define horiz_height 15
 static char horiz_bits[] = {
-   0x00, 0x00, 0x00, 0x00, 0xff, 0x7f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-   0x00, 0x00, 0xff, 0x7f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-   0xff, 0x7f, 0x00, 0x00, 0x00, 0x00};
+   (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0xff, (char)0x7f, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00,
+   (char)0x00, (char)0x00, (char)0xff, (char)0x7f, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00, (char)0x00,
+   (char)0xff, (char)0x7f, (char)0x00, (char)0x00, (char)0x00, (char)0x00};
--- a/src/gtk/verti.xbm
+++ b/src/gtk/verti.xbm
@@ -1,6 +1,6 @@
 #define verti_width 15
 #define verti_height 15
 static char verti_bits[] = {
-   0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10,
-   0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10, 0x84, 0x10,
-   0x84, 0x10, 0x84, 0x10, 0x84, 0x10};
+   (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10,
+   (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10,
+   (char)0x84, (char)0x10, (char)0x84, (char)0x10, (char)0x84, (char)0x10};
